make/include/versioning

```diff
@@ -25,7 +25,8 @@ GIT_TAG=${GIT_TAG:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $1 }' )}
 GIT_COMMITS=${GIT_COMMITS:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $2 }' )}
 GIT_SHA=${GIT_SHA:-$(echo ${GIT_DESCRIBE} | awk -F - '{ print $3 }' )}
 
-CF_VERSION=$(cat "${GIT_ROOT}/CF_VERSION")
+. $(dirname $(dirname "$0"))/bin/dev/versions.sh
```

Comment:
During my tests it found the correct file. So, looks to be ok.
I was actually uncertain and fully prepared to hunt down the syntax for the `$BASH_SOURCE` thing.
When it worked I shelved that.